### PR TITLE
Document revisions

### PR DESCRIPTION
**Pull request by kylewludwig**
- Updated power switch to 6A from 5A on vi.openxcplatform.com/electrical/design/power.html
- Changed from pin 8 to 9 on openxcplatform.com/vehicle-interface/hardware.html
- "servies and" typo corrected to "services are" in openxcplatform.com/android/tutorial.html
- Fixed bad link to python usb on openxcplatform.com/python/getting-started.html

### CODE REVIEW

File: vehicle-interface/firmware.mkd [hunk 1]
```diff
@@ -38,15 +35,14 @@ it.
 
 To read expanded data from a particular car, look for firmware distributed as
 pre-compiled binaries by your car's manufacturer. Check the [list of supported
-vehicles](/hardware/vehicles.html) for your car's make and model and for links
+vehicles](http://openxcplatform.com/hardware/vehicles.html) for your car's make and model and for links
```
Comment (emarsman) [inline vehicle-interface/firmware.mkd]: This should be a relative link. Why did you change it? It was working before.

Comment (kylewludwig) [reply vehicle-interface/firmware.mkd]: The relative link gave me a 404 error on Github. Should I make it a separate commit to the file or leave it out?

Update: checked it on the site - it works. I'll change it back

**Review by emarsman — APPROVED** [review]
